sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -222,4 +221,11 @@ internal class ViewAnnotationManagerImpl(
       currentViewsDrawnMap[it.identifier] = it.leftTopCoordinate
     }
   }
+
+  private inline fun <reified V> checkForError(expected: Expected<String, V>): V? {
```

Comment:
as return type is not Error maybe better to rename to `getValue` or `getNotError`

Reply:
renamed to `getValue`